antarest/study/storage/rawstudy/model/filesystem/lazy_node.py

```diff
@@ -121,14 +142,33 @@ def save(self, data: Union[str, bytes, S], url: Optional[List[str]] = None) -> N
                 self.config.path.unlink()
             return None
 
-        self.dump(cast(S, data), url)
+        self.dump(t.cast(S, data), url)
         if self.get_link_path().exists():
             self.get_link_path().unlink()
         return None
 
+    def move_file_to(
+        self, target: t.Union[Path, "LazyNode[t.Any, t.Any, t.Any]"], force: bool = False, copy: bool = False
+    ) -> None:
+        if isinstance(target, Path):
+            target.unlink(missing_ok=True)
+            if copy:
+                shutil.copy2(self.infer_path(), target)
+            else:
+                self.infer_path().rename(target)
+        else:
+            target_path = target.infer_target_path(self.infer_is_link_path())
+            if not force and target_path.exists():
+                raise FileExistsError(f"Target path {target_path} already exists")
+            target_path.unlink(missing_ok=True)
+            if copy:
+                shutil.copy2(self.infer_path(), target_path)
```

Comment:
Don't think we need any file's metadata so we should simply use `copy`

Reply:
ok